tests/hooks/use-suggestions.spec.ts

```diff
@@ -141,13 +146,12 @@ describe('use-suggestions', () => {
 
     describe('when the field has values defined', () => {
         it('suggestions should match the field values', () => {
+            const levelValues = [`'high'`, `'medium'`, `'low'`, 30]
             findSuggestions('Level = ')
-            expect(suggestions.value).toEqual([
-                `'high'`,
-                `'medium'`,
-                `'low'`,
-                30
-            ])
+            expect(suggestions.value).toEqual(levelValues)
+
+            findSuggestions('Level IN ')
+            expect(suggestions.value).toEqual(levelValues)
```

Comment:
please also test the case where we have `Level IN 'high',`
That should also show the level values too in suggestion ( in the future we want to filter out the selected values already but thats a nice to have )

Reply:
done